libs/src/sdk/api/chats/ChatsApi.ts

```diff
@@ -116,14 +129,18 @@ export class ChatsApi
     const query: HTTPQuery = {
       timestamp: new Date().getTime()
     }
-    if (requestParameters?.limit) {
-      query['limit'] = requestParameters.limit
+    const { limit, before, after } = parseRequestParameters(
+      'getAll',
+      ChatGetAllRequestSchema
+    )(requestParameters)
+    if (limit) {
+      query['limit'] = limit
     }
-    if (requestParameters?.before) {
-      query['before'] = requestParameters.before
+    if (before) {
+      query['before'] = before
```

Comment:
this + `query['after']` will be empty if `before`/`after`/ is 0, just checking that things would still work as expected?

Reply:
Yeah these should be strings/timestamps so 0 would throw at validation time